src/test/webview/suggestionProviders.test.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { expect } from 'chai';
+import { getSuggestions } from '../../helpers/SuggestionRegistry';
+
+suite('Channel API', () => {
+	suite('get Suggestions for OS environment variables', function () {
+		let originalEnv: NodeJS.ProcessEnv;
+
+		setup(() => {
+			originalEnv = { ...process.env };
+
+			// define test environment variables
+			process.env.TEST_ALPHA = 'A';
+			process.env.TEST_BETA = 'B';
+			process.env.TESTCASE = 'C';
+			process.env.Test_Mixed = 'mixed';
+			process.env.PATH = '/usr/local/bin';
```

Comment:
Do we have a test about writing `Te` and then getting `Tst_Mixed` first, and then the others?

Reply:
it was kinda covered, but I have improved second test to cover that more specifically - https://github.com/KaotoIO/vscode-kaoto/pull/973/files#diff-6d0299b63280917e832dd28caba420b15b5aef063be49b170fd12c8a96337f97R48